src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { LayoutComponent } from './layout/layout.component';
+import { TitleComponent } from './title/title.component';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule,
+  ],
+  exports: [
```

Comment:
Components this module exports.

If you want to use any of these components in another module you can import `SharedModule` (this module) in the other module, anything that is exported here will be available in the other end.